CHANGELOG.md

```diff
@@ -6,8 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Security
+- Updated all dependency versions in pom.xml and added maven-enforcer-plugin
+  [conjurdemos/pet-store-demo#52](https://github.com/conjurdemos/pet-store-demo/pull/52)
```

Comment:
```suggestion
  [conjurdemos/pet-store-demo#54](https://github.com/conjurdemos/pet-store-demo/pull/54)
```

Reply:
Thanks - I forgot issues take up the same sequence as PRs.  Just checked the closed PRs to guess the next number.